lib/services/project-service.ts

```diff
@@ -73,11 +73,10 @@ export class ProjectService implements IProjectService {
 	public isValidNativeScriptProject(pathToProject?: string): boolean {
```

Comment:
This method is created for Fusion, where when the user selects a folder, we try to check if this is a valid project.
All NativeScript CLI users will see the message from `initializeProjectData` method.